### ISSUE TITLE
.1 in name triggers different rules

### ISSUE THREAD

**Issue by jleach** [posabsolute/jQuery-Validation-Engine#126]
If I use a .1 at the end of a name= field in a checkbox validation then a different validation rule is triggered. Only the sequence that seems to affect this is .1; i've tested with .1,.2,.3,.4,.5.

For example:

```
<input name='input_5.1' class='validate[required] checkbox' type='checkbox' id='choice_5_1' />
Triggers  : "alertTextCheckboxMultiple": "* Please select an option",
Expected: "alertTextCheckboxe": "* This checkbox is required",

<input name='input_5.4' class='validate[required] checkbox' type='checkbox' id='choice_5_1' />
Triggers:  "alertTextCheckboxe": "* This checkbox is required",
Expected: "alertTextCheckboxe": "* This checkbox is required",
```

Environment:
Browser: Google Chrome 12.0.742.122
jquery: 1.6.2-min
